### PR TITLE
Adds icons to top level items

### PR DESCRIPTION
**Pull request by oliverviljamaa**
# v0.7.0
## Adds icons to first level items

Only the used icons are included in the resulting bundle, as inline SVG-s.

### CODE REVIEW

File: src/Navigation/Menu/Items/Item/Item.js [hunk 1]
```diff
@@ -3,29 +3,36 @@ import Types from 'prop-types';
...
+    const { text, link, Icon, items } = this.props;
+
     return (
       <li
         ref={element => {
           this.itemDOMElement = element;
         }}
       >
-        <a href={this.props.link}>{this.props.text}</a>
+        <a href={link}>
```
Comment (nabaroa) [inline src/Navigation/Menu/Items/Item/Item.js]: The icons are visible also in desktop and shouldn´t. With visible-xs-inline-block visible-sm-inline-block classes we can hide them from desktop.

File: src/Navigation/Menu/Items/Item/Item.js [hunk 2]
```diff
@@ -3,29 +3,36 @@ import Types from 'prop-types';
...
+          <Icon className="tw-public-navigation-item__icon m-r-2" />
```
Comment (nabaroa) [inline src/Navigation/Menu/Items/Item/Item.js]: The icons are visible also in desktop and shouldn´t. With visible-xs-inline-block visible-sm-inline-block classes we can hide them from desktop.

Comment (oliverviljamaa) [reply src/Navigation/Menu/Items/Item/Item.js]: Thanks, good catch. The `.less` file works, but `@transferwise/icons` wasn't published and the old version overrides the `display`. I'll use the utility classes instead for brevity, though!

File: src/items/items.js [hunk 3]
```diff
@@ -19,9 +19,9 @@ export function getButtonItemInLanguage(language) {
   return translateItem(buttonItem, language);
 }
 
-function translateItem({ translationKey, link }, language) {
+function translateItem({ translationKey, ...item }, language) {
   return {
     text: translate(translationKey, language),
-    link,
+    ...item,
```
Comment (StevePole) [inline src/items/items.js]: This syntax is blowing my mind.

**Pull request merged**